src/pudl/metadata/resources/ferc1.py

```diff
@@ -731,7 +736,9 @@
         "field_namespace": "ferc1",
     },
     "core_pudl__assn_ferc1_dbf_pudl_utilities": {
-        "description": "This table maps the assign utility ID FERC1 to the native utility ID from the FERC1 DBF inputs - originally reported as respondent_id.",
+        "description": {
+            "additional_summary_text": "PUDL FERC1 utility IDs and the native FERC1 DBF utility IDs originally reported as ``respondent_id``.",
```

Comment:
recognizing that these noun phrases are gnarly. while our PUDL naming conventions put the source last in the variable name, very few descriptors are permitted to come after the noun in regular english, and we're not in attorney general or battle royale territory here. So, before the noun they all must go. I tried to be consistent though:

PUDL FERC1 utility ID

* PUDL picked it
* it's not just a utility that's being identified, it's a _FERC1_ utility

FERC1 DBF utility ID

* FERC1 picked it
* it's not just a utility that's being identified, it's a utility _from a DBF_

& similarly for 
* FERC1 XBRL utility ID
* PUDL FERC714 respondent ID
* FERC714 CSV respondent ID
* FERC714 XBRL respondent ID

Reply:
I like what you've done i think the standardization makes it much more clear and I appreciate the source in front of the variable name in the text.

The one suggestion I would make would be to use "PUDL assigned" or "PUDL derived". I personally think that makes it more clear why there are two source names for say "PUDL FERC1". but not a hill i will die on